src/test/java/org/jabref/logic/shared/DBMSSynchronizerTest.java

```diff
@@ -71,6 +71,7 @@ public void setup() throws Exception {
 
     @AfterEach
     public void clear() throws SQLException {
+        dbmsSynchronizer.closeSharedDatabase();
         this.dbmsConnection.getConnection().close();
```

Comment:
You can remove this statement as the dbmsSynchronizer uses the same connection, doesn't it?

Reply:
If you mean the statement `this.dbmsConnection.getConnection().close();`, then yes.